ui/platform/linux/ui_linux_wayland_integration_dummy.cpp

```diff
@@ -17,6 +17,8 @@ struct WaylandIntegration::Private {
 
 WaylandIntegration::WaylandIntegration() {
 }
+	
+WaylandIntegration::~WaylandIntegration() = default;
```

Comment:
You defined the destructor with C++11 `= default;` new syntax, however above, default constructor was already defined in old-fashioned way by empty braces `{}`. It remains a bit inconsistent. :disappointed:

Reply:
It is copy-paste from non-dummy implementation in
https://github.com/kvtb/lib_ui/blob/d8f436d9899cd6255180003ef948584f8a2cc63c/ui/platform/linux/ui_linux_wayland_integration.cpp#L72